tests/unit/components/frost-bunsen-form/component-test.js

```diff
@@ -0,0 +1,109 @@
+import {expect} from 'chai'
+import {setupComponentTest} from 'ember-mocha'
+import {afterEach, beforeEach, describe, it} from 'mocha'
+import sinon from 'sinon'
+
+describe('Unit: frost-bunsen-form', function () {
+  setupComponentTest('frost-bunsen-form', {
+    unit: true
+  })
+
+  let component, sandbox
+
+  beforeEach(function () {
+    sandbox = sinon.sandbox.create()
+    component = this.subject({
+      bunsenModel: {},
+      bunsenView: {},
+      onChange () {},
+      onError () {}
+    })
+    component.init()
```

Comment:
You shouldn't need to call `init` here as `this.subject` calls it indirectly through instantiation.